src/options.h

```diff
@@ -57,12 +58,23 @@ void set_default_values_from_env()
             rotated_files = 5;
         }
     }
+
+    if ( log_directory ==  NULL ) {
+        env_val = g_getenv("LOGPROXY_LOG_DIRECTORY");
+        if ( env_val != NULL ) {
+            log_directory = (gchar *)env_val;
+        }
+        else {
+            log_directory = g_strdup(g_get_current_dir());
```

Comment:
read on `g_get_current_dir` doc:

> The returned string should be freed when no longer needed.

So we should leak some memory here (ok it's anecdotal)

```suggestion
            log_directory = g_get_current_dir();
```